content/docs/2.9/operate/prometheus.md

```diff
@@ -10,6 +10,7 @@ weight = 200
 
 The KEDA Operator exposes Prometheus metrics which can be scraped on port `8080` at `/metrics`. The following metrics are being gathered:
 
+- `keda_operator_cr_totals` - Total number of KEDA custom resources per namespace for each custom resource type (CRD).
```

Comment:
Here I am again to be a pain but `cr` is really a bad name.

Can we please rename it to `keda_operator_resource_totals` or `keda_operator_customresource_totals` please?
```suggestion
- `keda_operator_resources_totals` - Total number of KEDA custom resources per namespace for each custom resource type (CRD).
```

That way people know what it means when not looking at this page